monday/client.py

```diff
@@ -7,23 +7,36 @@
 """
 
 from .__version__ import __version__
-from .resources import ItemResource, UpdateResource, TagResource, BoardResource, UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource
+from .resources import CustomResource, ItemResource, ColumnsResource, UpdateResource, TagResource, BoardResource, \
+    UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource, MeResource
+
+_DEFAULT_HEADERS = {
+    "API-Version": "2023-10"
+}
 
 
 class MondayClient:
-    def __init__(self, token):
+    def __init__(self, token, headers=None):
```

Comment:
[not blocking] seems like we're not using typing consistently in this project. 
We should probably put up another PR at some point to address that.